content/docs/querying/operators.md

```diff
@@ -17,7 +23,7 @@ The following binary arithmetic operators exist in Prometheus:
 Binary arithmetic operators are defined between scalar/scalar, vector/scalar,
 and vector/vector value pairs.
 
-**Between two scalars**, the behavior is obvious: they evaluate to another
+**Between two scalars**, the behaviour is obvious: they evaluate to another
```

Comment:
http://grammarist.com/usage/ie-eg/
Doesn't seem super strict, though. But I also like the Oxford comma - so I guess going British with commas and US with words is fine. There are also some differences regarding width of dashes and whether or not they are surrounded by spaces.

Usage of full phrases noted.